database/migrations/2019_09_28_171753_config_fix.php

```diff
@@ -0,0 +1,577 @@
+			],
+			[
+				'key' => 'landing_flickr',
+				'value' => 'https://www.flickr.com//JohnSmith',
+				'cat' => 'Mod Welcome',
+				'type_range' => STRING,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'landing_twitter',
+				'value' => 'https://www.twitter.com//JohnSmith',
+				'cat' => 'Mod Welcome',
+				'type_range' => STRING,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'landing_instagram',
+				'value' => 'https://instagram.com/JohnSmith',
+				'cat' => 'Mod Welcome',
+				'type_range' => STRING,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'landing_youtube',
+				'value' => 'https://www.youtube.com//JohnSmith',
+				'cat' => 'Mod Welcome',
+				'type_range' => STRING,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'landing_background',
+				'value' => 'dist/cat.jpg',
+				'cat' => 'Mod Welcome',
+				'type_range' => STRING,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'thumb_2x',
+				'value' => '1',
+				'cat' => 'Image Processing',
+				'type_range' => BOOL,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'small_2x',
+				'value' => '1',
+				'cat' => 'Image Processing',
+				'type_range' => BOOL,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'medium_2x',
+				'value' => '1',
+				'cat' => 'Image Processing',
+				'type_range' => BOOL,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'site_title',
+				'value' => 'Lychee v4',
+				'cat' => 'config',
+				'type_range' => STRING,
+				'confidentiality' => '0',
+			],
+			[
+				'key' => 'site_copyright_enable',
+				'value' => '1',
+				'cat' => 'config',
+				'type_range' => BOOL,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'site_copyright_begin',
+				'value' => '2019',
+				'cat' => 'config',
+				'type_range' => INT,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'site_copyright_end',
+				'value' => '2019',
+				'cat' => 'config',
+				'type_range' => INT,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'api_key',
+				'value' => '',
+				'cat' => 'Admin',
+				'type_range' => STRING,
+				'confidentiality' => '3',
+			],
+			[
+				'key' => 'allow_online_git_pull',
+				'value' => '1',
+				'cat' => 'Admin',
+				'type_range' => BOOL,
+				'confidentiality' => '3',
+			],
+			[
+				'key' => 'force_migration_in_production',
+				'value' => '0',
+				'cat' => 'Admin',
+				'type_range' => BOOL,
+				'confidentiality' => '3',
+			],
+			[
+				'key' => 'additional_footer_text',
+				'value' => 'config',
+				'cat' => 'config',
+				'type_range' => STRING,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'display_social_in_gallery',
+				'value' => '0',
+				'cat' => 'config',
+				'type_range' => BOOL,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'public_search',
+				'value' => '0',
+				'cat' => 'config',
+				'type_range' => BOOL,
+				'confidentiality' => '0',
+			],
+			[
+				'key' => 'gen_demo_js',
+				'value' => '1',
```

Comment:
What's the value of having it enabled for regular users?

Reply:
Good point.

Reply:
I think it was disabled, it's just that I copied the table from my test DB... :'D